app/gateway/2.6.x/install-and-run/docker.md

```diff
@@ -57,53 +51,91 @@ docker pull kong:{{page.kong_versions[page.version-index].ce-version}}-alpine
 ```
 {% endnavtab %}
 {% endnavtabs %}
-{% endcapture %}
-{{ pull_image | indent | replace: " </code>", "</code>" }}
 
-   You should now have your {{site.base_gateway}} image locally.
+You should now have your {{site.base_gateway}} image locally.
 
-2. Tag the image:
+Next, choose a path and install Kong Gateway:
```

Comment:
Why would I choose one over the other?

Reply:
This question is much bigger on the inside, and really we should be linking them to a comparison of database-backed vs db-less deployment options (which... currently doesn't exist? We have a very top-level one that Shane asked us to remove because it wasn't super accurate, and no one's written one since.) 

In lieu of a proper breakdown, I'll try to come up with a one sentence summary of each option here.

Reply:
If the next step is to follow the getting started guide which needs the admin API to be writable, maybe we just add "We recommend trying out Kong with a DB if this is your first time trying Kong"

Reply:
The getting started guide actually provides yaml config snippets and gives instructions to apply them using decK - so this isn't a limitation.